cli/lib/kontena/cli/config.rb

```diff
@@ -103,6 +103,10 @@ def load_settings_from_config_file
         end
 
         self.current_server = ENV['KONTENA_MASTER'] || settings['current_server']
+        unless find_server(self.current_server)
+          warn "Master '#{self.current_server}' not found in configuration"
```

Comment:
Should this be an `abort` instead of just `warn`?

```
$ KONTENA_MASTER=terom-test bundle exec bin/kontena node list
Master 'terom-test' not found in configuration
 [error] ArgumentError : You are not logged into a Kontena Master. Use: kontena master login
         See /home/kontena/.kontena/kontena.log or run the command again with environment DEBUG=true set to see the full exception
```